src/core/server/saved_objects/migrationsv2/integration_tests/type_registrations.test.ts

```diff
@@ -71,7 +71,7 @@ const previouslyRegisteredTypes = [
   'ml-telemetry',
   'monitoring-telemetry',
   'osquery-saved-query',
-  'osquery-usage-metric',
+  'osquery-manager-usage-metric',
```

Comment:
@patrykkopycinski if we released a Kibana version with this SO type name then we need to add the old name to the array here to delete the old objects when the user upgrades: https://github.com/elastic/kibana/blob/master/src/core/server/saved_objects/migrations/core/elastic_index.ts#L34

Reply:
changed, Thank you @joshdover!

Reply:
@patrykkopycinski You know what I forgot to mention that I think the old type name actually needs to be in both arrays or a test is going to fail that ensures that all items in `REMOVED_TYPES` is also contained in this `previouslyRegisteredTypes` array.